src/components/common/ConnectWallet/AccountCenter.tsx

```diff
@@ -14,17 +13,13 @@ import ChainSwitcher from '../ChainSwitcher'
 import useAddressBook from '@/hooks/useAddressBook'
 import { type ConnectedWallet } from '@/hooks/wallets/useOnboard'
 import WalletInfo, { UNKNOWN_CHAIN_NAME } from '../WalletInfo'
-import { getShortName } from '@/utils/chains'
 
 const AccountCenter = ({ wallet }: { wallet: ConnectedWallet }) => {
   const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(null)
   const onboard = useOnboard()
   const chainInfo = useAppSelector((state) => selectChainById(state, wallet.chainId))
   const addressBook = useAddressBook()
-
-  const prefix = useMemo(() => {
-    return chainInfo?.shortName || (wallet?.chainId && getShortName(wallet?.chainId))
-  }, [chainInfo?.shortName, wallet?.chainId])
+  const prefix = chainInfo?.shortName
```

Comment:
Do we not want to show the prefixes of networks not in the chain configs but in `chains`? See [here](https://github.com/safe-global/web-core/pull/1606#issuecomment-1410229573).

Reply:
It will just an address w/o any prefix, it's fine.